docs/faq/code-analysis/which-metrics-does-codacy-calculate.md

```diff
@@ -59,7 +59,7 @@ Codacy calculates the number of issues in the following static code analysis cat
 -   **Documentation:** Methods and classes that don't have the correct comment annotations
 <!--issue-categories-end-->
 
-Besides this, Codacy also allows you to compare issues across repositories with different sizes by calculating the **percentage of issues relative to an expected baseline** of 100 issues with an average cost of 10 per 1000 lines of code, where the cost of each issue depends on its severity: Critical = 10, Medium = 5, Minor = 1.
+Besides this, Codacy also allows you to compare issues across repositories with different sizes by calculating the **issue cost relative to an expected baseline of 10 points per 1000 lines of code**, where the cost of each issue depends on its severity: Critical = 10 points, Medium = 5 points, Minor = 1 point. This means that if your repository has 50% issues, the amount and severity of the issues in your repository is half of the expected baseline.
```

Comment:
Thanks, @prcr! I suggest a couple of tweaks:

[Your calculations led us to the conclusion](https://github.com/codacy/docs/pull/1657#discussion_r1145932404) that it's most likely 1000 points per 1000 lines. We could express this in two ways:
* `1000 points per 1000 lines of code` (or any arbitrary round number number, such as 100 points per 100 lines of code)
* `1 point per line of code`

I would also remove the word "expected", since this issue density would be alarming in an actual codebase. 

```suggestion
Besides this, Codacy also allows you to compare issues across repositories with different sizes by calculating the **issue cost relative to a baseline of 1000 points per 1000 lines of code**, where the cost of each issue depends on its severity: Critical = 10 points, Medium = 5 points, Minor = 1 point. This means that if your repository has 50% issues, the amount and severity of the issues in your repository is half of the expected baseline.
```

Reply:
Indeed, this is a nice simplification! I'd actually go with `1 point per line of code`, as it's more elegant. :nerd_face:

Reply:
@nicklem, I applied your suggestion with a couple more minor tweaks.

I think that the simplified explanation that we have now makes more sense than before since there are less "magic numbers" involved. :+1: